scavenger-agent-java/build.gradle.kts

```diff
@@ -72,6 +73,40 @@ dependencies {
     testImplementation("org.mockito:mockito-inline:4.3.1")
 }
 
+testSets {
+    create("integrationTest")
+}
+
+dependencies {
+    "integrationTestImplementation"("org.springframework.boot:spring-boot-starter-test:2.5.12")
+    "integrationTestImplementation"("org.springframework.boot:spring-boot-starter-aop:2.5.12")
+    "integrationTestImplementation"("com.github.tomakehurst:wiremock:2.27.2")
+    "integrationTestImplementation"("org.grpcmock:grpcmock-junit5:0.9.4")
+}
+
+fun javaPaths(vararg versions: Int) = versions.joinToString(",",
+    transform = { version: Int ->
+        "$version:" + javaToolchains.launcherFor {
+            languageVersion.set(JavaLanguageVersion.of(version))
+        }.get().executablePath
+    })
+
+val integrationTestRuntimeClasspath = configurations.named("integrationTestRuntimeClasspath").get().asPath
+
+tasks.named<Test>("integrationTest") {
```

Comment:
I've made integration test always run when `scavenger-agent-java` is built in [e42798c](https://github.com/naver/scavenger/pull/71/commits/e42798c2c90a6fe392a79f72530a48955be56371).
On the other hand, I also think it shouldn't be run all the time when it's built because it costs a fair amount of time.
What is your opinion on this? Should we only run integration test when there's a PR against the develop branch?

Reply:
It's been added to the check task. I missed that.
Good :)
https://github.com/naver/scavenger/actions/runs/5173676650/jobs/9319166892?pr=71#step:5:82
<img width="385" alt="image" src="https://github.com/naver/scavenger/assets/27491403/da28cafc-8ba6-4246-b584-f0bc9d6c9997">